prototypes/process-explorer/dotnet/src/MorganStanley.ComposeUI.ProcessExplorer.Server/Server/Abstractions/ProcessExplorerBuilderExtensions.cs

```diff
@@ -26,6 +26,13 @@ public static class ProcessExplorerBuilderExtensions
         if (options != null) builder.ServiceCollection.Configure(options);
 
         builder.ServiceCollection.AddGrpc();
+        builder.ServiceCollection.AddCors(o => o.AddPolicy("AllowAll", builder =>
+        {
+            builder.AllowAnyOrigin()
```

Comment:
These configurations might be OK in a prototype, but definitely not OK for production due to security concerns. See https://learn.microsoft.com/en-us/aspnet/core/security/cors?view=aspnetcore-8.0#set-the-allowed-origins for details.

Reply:
Noted!